pkg/server/commands/connectedobjects/connected_objects.go

```diff
@@ -158,20 +168,24 @@ type ConnectedObjectsResult struct {
 	ResultStatus ConditionalResultStatus
 }
 
-func (c *ConnectedObjectsQuery) execute(
+// Execute yields all the objects of the provided objectType that the given user has, possibly, a specific relation with
+// and sends those objects to resultChan. It guarantees no duplicate objects sent.
```

Comment:
If we're guaranteeing no duplicate objects sent here, why are we doing the same in `commands/list_objects.go`?

Reply:
good point, i think ListObjects doesn't need to have a map anymore 🤔

Reply:
Mind creating the issue and cross referencing with this PR to track?

Reply:
I'll change in this PR